core/order.go

```diff
@@ -741,6 +781,19 @@ func (n *OpenBazaarNode) CalcOrderId(order *pb.Order) (string, error) {
 	return id.B58String(), nil
 }
 
+func (n *OpenBazaarNode) getMarketPriceInSatoshis(symbol string, quantity uint32) (uint64, error) {
+	if n.ExchangeRates == nil {
+		return 0, ErrMarketPriceRequiresExchangeRates
+	}
```

Comment:
If this can be nil, the other call `getPriceInSatoshi` could also crash if ExchangeRates are off. Can we improve the other method to include this check?